rfcs/0000-add-host-fields.md

```diff
@@ -0,0 +1,128 @@
+# 0000: Name of RFC
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
```

Comment:
Feel free to remove these HTML comments as you go along. E.g. this stage 0 requirement is addressed by the paragraph below, so you can remove this.

At stage 1, you can remove each stage 1 comment as you address them.

Reply:
Done. Thank you!